cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -151,10 +165,12 @@ def _get_chunk_metadata(study_area_metadata: dict, chunk_id: str) -> dict:
         or chunk_metadata.get("col_count") is None
         or chunk_metadata.get("x_ll_corner") is None
         or chunk_metadata.get("y_ll_corner") is None
+        or chunk_metadata["x_index"] is None
```

Comment:
I think here (and above) you mean `"x_index" not in chunk_metadata` as the `["x_index"]` access would throw an exception if it's not present.

Reply:
woops good catch